c_src/erasure.c

```diff
@@ -67,7 +67,7 @@ encode(ErlNifEnv * env, int argc, const ERL_NIF_TERM argv[])
     int bytes_per_shard = input.size / k;
     int extra_bytes = input.size % k;
 
-    char *shards = calloc(k+m, blockspacing);
+    char *shards = calloc(k+m, blockspacing-1);
```

Comment:
Really? An off by one error?!

Reply:
I added that on purpose to trigger an error. I'll rebase without that commit before merging.